### ISSUE TITLE
♿️ a11y(bal-carousel): aria-hidden

### ISSUE THREAD

**Issue by GinaBiondo** [baloise/design-system#945]
### Current Behavior

Currently aria-hidden is not set correctly.

### Expected Behavior

aria-hidden should be set according to the elements visibility on screen. Also the tabindex should be checked with that.

### Steps to Reproduce

Carousel and all components, that use it.

### Version

13.4.4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hirsch88** [comment]
@GinaBiondo-aperto do we really need that?

Because event [swiperjs](https://swiperjs.com/) does not do that

**Comment by hirsch88** [comment]
will not be done, due to the fact that a user will not get any information out of a slide like in the product slider
